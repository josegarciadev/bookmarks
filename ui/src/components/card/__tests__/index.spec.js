import { shallow } from 'enzyme';

import { Card } from '../index';

describe('Card', () => {
    it('should match snapshot', () => {
        const rendered = shallow(<Card />);
        expect(rendered).toMatchSnapshot();
    });
});