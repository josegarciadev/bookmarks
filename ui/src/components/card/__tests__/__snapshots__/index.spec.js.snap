// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<div
  className="card"
>
  <p
    className="card-label"
  />
</div>
`;
